lms/djangoapps/branding/__init__.py

```diff
@@ -17,12 +17,16 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
 
-def get_visible_courses():
+def get_visible_courses(org=None):
     """
     Return the set of CourseDescriptors that should be visible in this branded instance
     """
-    filtered_by_org = microsite.get_value('course_org_filter')
-    courses = CourseOverview.get_all_courses(org=filtered_by_org)
+    microsite_org = microsite.get_value('course_org_filter')
+
+    orgs = [org, microsite_org]
+    orgs = [org for org in orgs if org is not None]
+
+    courses = CourseOverview.get_all_courses(orgs=orgs)
```

Comment:
@rlucioni sorry, the set terminology i used in my original comment was wrong - i agree that when the query org and the microsite org are both nonempty and not the same, that we should not return all visible runs.

The scenario I remain concerned about is when this API is being used in the context of a microsite, and the caller passed some org other than that of the microsite, the query org will effectively be ignored.  i think it would make more sense to have the results be empty in that case, since the caller asked for course runs for an org that doesn't exist in the microsite context.